lightning/src/routing/router.rs

```diff
@@ -784,6 +800,8 @@ where L::Target: Logger {
 					// Verify the liquidity offered by this channel complies to the minimal contribution.
 					let contributes_sufficient_value = available_value_contribution_msat >= minimal_value_contribution_msat;
 
+
+
```

Comment:
nit: remove extra blank lines

Reply:
Done in https://github.com/lightningdevkit/rust-lightning/commit/60b4bb75dfe4901ad89171e7e4ff3272097fc7e9.